RELEASE.md

```diff
@@ -63,1332 +51,3 @@
 
 *   Deprecated python3.6 support.
 
-# Version 0.34.1
```

Comment:
Why is the previous history being removed?

Reply:
We Publish release log information only to that particular release (removing previous release notes ). For example https://github.com/tensorflow/model-analysis/blob/r0.34.0/RELEASE.md